site/content/en/blog/releases/v0.6.0.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "v0.6.0"
+linkTitle: "v0.6.0"
+date: 2020-09-01
+description: >
+  Cloud Provider Azure v0.6.0
+---
+
+## Major changes since v0.5.0
+
+-------
+
+- Increase the e2e test coverage for cluster autoscaler([#364](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/364))
+- Use hugo to generate doc website([#358](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/358))
+- Update E2E test related docs and script([#355](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/355))
+- Update vendor against release 1.18([#354](https://github.com/kubernetes-sigs/cloud-provider-azure/pull/354))
```

Comment:
this should be 1.19

Reply:
release-1.18 updates is part of v0.5.1

Reply:
we have not done such an operation to update vendor against release 1.19.

Reply:
will update it now